src/explorer/CosmosDBTreeItem.ts

```diff
@@ -132,6 +132,16 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
                 await createdDatabase.cosmosExtensionItem.reveal();
             }
         });
+        if (createdDatabase.cosmosExtensionItem.azureData && createdDatabase.cosmosExtensionItem.connectionString.startsWith('postgres://')) {
+            const openInPortal: vscode.MessageItem = { title: 'Open in Portal' };
+            const manageFirewallMessage: string = localize('manageFirewallRulesMsg', `Manage Firewal Rules for the new Database connection "${createdDatabase.label}".`);
```

Comment:
When you localize strings, you can't use template strings like this.  Instead, it should be like this:

` const manageFirewallMessage: string = localize('manageFirewallRulesMsg', 'Manage Firewal Rules for the new Database connection "{0}".', createdDatabase.label);`